app/assets/javascripts/admin/models/web-hook.js.es6

```diff
@@ -0,0 +1,98 @@
+import RestModel from 'discourse/models/rest';
+import { default as computed, observes } from 'ember-addons/ember-computed-decorators';
+
+export default RestModel.extend({
+  // a better way to sideload the list?
+  WILDCARD_EVENT: {
+    id: 1,
+    name: 'wildcard'
+  },
+  EVENTS: Em.A([{
+    id: 2,
+    name: 'topic'
+  }, {
+    id: 3,
+    name: 'post'
+  }, {
+    id: 4,
+    name: 'user'
+  }]),
+
+  contentTypes: Em.A([{
+    id: 1,
+    name: 'application/json'
+  }, {
+    id: 2,
+    name: 'application/x-www-form-urlencoded'
+  }]),
+  content_type: 1, // prefer json,
+  web_hook_event_types: Em.A([{ // default event type
+    id: 3,
+    name: 'post'
+  }]),
```

Comment:
Ah ok I think I didn't get what you mean previously. Hardcoding all the type here is bad.

You can probably do something like this in your controller

```
    render json: {
      web_hooks: serialize_data(WebHook.all, AdminWebHookSerializer)
      web_hook_event_type: serialize_data(WebHookEventType.all, some_serializer)
      other models that you need
    }
```